judgels-backends/jophiel/jophiel-dist/init.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ex
+
+COMMAND=$1
+
+if [[ -z ${COMMAND} ]]; then
+	COMMAND="console"
+fi
+
+if [[ "$COMMAND" == "console" ]]; then
```

Comment:
- if the command is `dbMigrate`, pass it to the binary
- else, pass it to the original init.sh

because there are other commands than `console`; we must still support it